src/@rocketseat/gatsby-theme-docs/components/Layout/index.js

```diff
@@ -1,6 +1,6 @@
 /* @jsx jsx */
 import { useState, useRef } from 'react'
-import { jsx, css } from '@emotion/core'
+import { jsx, css } from '@emotion/react'
```

Comment:
whats the purpose of this? does everything work without it?

Reply:
I believe the JSX pragma was set in this file for `jsx` construction to provide syntactic sugar for `React.createElement` function and utilize the props.